deployment/refinery-modules/refinery/manifests/solr.pp

```diff
@@ -39,10 +39,19 @@
       require => [ File["/opt"], Package['java'] ],
     }
     ->
-    file_line { "solr_config_home":
-      path  => "/var/solr/solr.in.sh",
-      line  => "SOLR_HOME=${django_root}/solr",
-      match => "^SOLR_HOME",
+    file_line {
+      "solr_config_home":
+        path  => "/var/solr/solr.in.sh",
+        line  => "SOLR_HOME=${django_root}/solr",
+        match => "^SOLR_HOME";
+      'solr_config_heap':
```

Comment:
Double quotes for consistency?

Reply:
Actually, they should all be single quotes unless variable interpolation required :) https://docs.puppet.com/puppet/5.1/style_guide.html#quoting

Reply:
Okay cool. Could you update the other double quotes w/o interpolated vars?

Reply:
In all the manifests? :)

Reply:
If you're feeling up to it I guess? But I was just thinking about ones from this diff.